spamwatch/client.py

```diff
@@ -163,3 +163,13 @@ def delete_ban(self, user_id: int) -> None:
         """Remove a ban"""
         self._make_request(f'banlist/{user_id}', method='delete')
     # endregion
+    
+    #region Stats
```

Comment:
this needs a space after the # to conform with PEP 8 and so its a valid region for PyCharm, same for the endregion comment